code/game/objects/items/devices/mat_synth.dm

```diff
@@ -243,10 +221,7 @@ var/static/list/mat2type = list(
 
 /obj/item/device/material_synth/examine(mob/user)
 	..()
-	if(istype(src, /obj/item/device/material_synth/robot))
-		to_chat(user, "It's been set to draw power from a power cell.")
-	else
-		to_chat(user, "It currently holds [matter]/[MAX_MATSYNTH_MATTER] matter-units.")
+	to_chat(user, "[istype(src, /obj/item/device/material_synth/robot) ? "It's been set to draw power from a power cell." : "It currently holds [matter]/[MAX_MATSYNTH_MATTER] matter-units."]")
```

Comment:
This is less readable than the initial version.

Reply:
changed back now